lib/src/rules/cascade_invocations.dart

```diff
@@ -128,20 +128,19 @@ class CascadeInvocations extends LintRule implements NodeLintRule {
 /// A CascadableExpression is an object that is built from an expression and
 /// knows if it is able to join to another CascadableExpression.
 class _CascadableExpression {
-  static final nullCascadableExpression = _CascadableExpression._internal(
-      null, [],
-      canJoin: false, canReceive: false, canBeCascaded: false);
+  static final nullCascadableExpression =
+      _CascadableExpression._internal(null, [], canReceive: false);
 
   /// Whether this expression can be joined with a previous expression via a
   /// cascade operation.
-  final bool? canJoin;
+  final bool canJoin;
 
   /// Whether this expression can receive an additional expression with a
   /// cascade operation.
   ///
   /// For example, `a.b = 1` can receive, but `a = 1` cannot receive.
   final bool? canReceive;
```

Comment:
Hopefully we can get rid of this one eventually too.

Reply:
Ah.  I missed that one.  Fixed now.  Thanks!